src/AspNetCoreModuleV1/AspNetCore/aspnetcoremodule.rc

```diff
@@ -75,10 +75,10 @@ BEGIN
     BEGIN
         BLOCK "040904b0"
         BEGIN
-            VALUE "CompanyName", "Microsoft"
+            VALUE "CompanyName", "Microsoft Corporation"
             VALUE "FileDescription", FileDescription
             VALUE "FileVersion", FileVersionStr
-            VALUE "InternalName", "aspnetcore.dll"
+            VALUE "InternalName", "aspnetcore"
```

Comment:
It would modify the dll file information. We recently added the commit hash to the file: https://github.com/aspnet/IISIntegration/commit/8dbe56426333c908a3b7263557aa43fba585f6b9. These are simply keeping values consistent. I don't see any reason this isn't a safe operation.